.github/workflows/deploy.yml

```diff
@@ -0,0 +1,47 @@
+name: Deploy the Website
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.SECRET_ACCESS_KEY }}
+          aws-region: ap-south-1
+
+      - name: Install modules
+        run: npm ci
+
+      - name: Run prettier
+        run: npx prettier -c .
+
+      - name: Build application
+        run: npm run build
+
+      - name: Craete zip file
+        run: |
+          apt-get update && apt-get install -y zip sed
+          npm run build
+          zip canopas_blog.zip ./dist/
+
+      - name: Artifacts zip file
+        uses: actions/upload-artifact@v3
+        with:
+          name: dist-with
+          path: canopas_blog.zip
+
+      - name: Upload to S3
+        run: |
+          - S3_FILE_NAME="canopas_blog_${CI_PIPELINE_IID}.zip"
```

Comment:
Is `CI_PIPELINE_IID` available on github actions? Search on it

Reply:
no, but [GITHUB_RUN_ID](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables) is available

Reply:
Have to use it instead of `CI_PIPELINE_IID`